### PR TITLE
Message added in Favourites fragment

### LINKED ISSUE #698: Add more information on favourites page
Screenshot 2019-04-11 at 3 29 48 PM

### Steps to reproduce
Select heart icon on title bar (on top right)

### Expected behaviour
Add a message below the animation "Select + icon to add your favourite features!"

### PR DESCRIPTION
**Pull request by rob729**
# Description

2ycajr

Fixes #698

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by rob729** [comment]
@vyankatesh24 @Swati4star Please review this PR.

File: app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java [hunk 1]
```diff
@@ -188,6 +191,7 @@ private void checkFavs(SharedPreferences sharedPreferences) {
         // if there are no favourites then show favourites animation
```
Comment (vyankatesh24) [inline app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java]: Update comment!

File: app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java [hunk 2]
```diff
@@ -256,6 +260,7 @@ private void viewVisibility(View view, String id) {
             mDoesFavouritesExist = true;
             // & disable favourites animation
```
Comment (vyankatesh24) [inline app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java]: Update comment

**Comment by rob729** [comment]
@vyankatesh24 I have done the suggested changes.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -507,5 +507,6 @@
     <!--favourite fragment-->
     <string name="favourites">Favourites</string>
     <string name="add_to_favourite">Add to Favourite</string>
+    <string name="favourites_text">Select + icon to add your favourite features!</string>
```
Comment (vyankatesh24) [inline app/src/main/res/values/strings.xml]: Add this string in other string locale files.

**Comment by rob729** [comment]
@vyankatesh24 @srv-twry I have done the suggested changes.

**Comment by vyankatesh24** [comment]
@rob729 checks are failing, have a look!

**Comment by rob729** [comment]
@vyankatesh24 Can you please tell me what could be the possible reason for this?

**Comment by vyankatesh24** [comment]
> @vyankatesh24 Can you please tell me what could be the possible reason for this?

 /home/travis/build/Swati4star/Images-to-PDF/app/src/main/res/values-fr/strings.xml:413:5-111: AAPT: error: unescaped apostrophe in string
      
  "Sélectionnez l'icône + pour ajouter vos fonctionnalités préférées!".
  /home/travis/build/Swati4star/Images-to-PDF/app/src/main/res/values-fr/strings.xml:413:5-111: AAPT: error: not a valid string.

**Comment by rob729** [comment]
@vyankatesh24 Thanks for the help.
Please review the PR.

**Pull request merged**